src/components/floating-menu/floating-menu.js

```diff
@@ -169,6 +183,7 @@ class FloatingMenu extends mixin(createComponent, eventedShowHideState) {
       }
       this._getContainer().appendChild(this.element);
       this._place();
+      (this.element.querySelector(this.options.selectorPrimaryFocus) || this.element).focus();
```

Comment:
Below shows how it works without `data-floating-menu-primary-focus`:

![floating-menu-without-primary-focus](https://user-images.githubusercontent.com/1259051/28394591-1211e5e0-6d29-11e7-87ac-e262bfc2acd0.gif)